.gitignore

```diff
@@ -25,3 +25,4 @@ api/*/*.pb.go
 .*.swp
 .*.swx
 
+tilt_modules
```

Comment:
the intent is that this directory should be committed, so that everyone in the repo is using the same version of the extension (https://docs.tilt.dev/extensions.html#commit-to-source-control)